lib/nrf5/common/uart_common.c

```diff
@@ -0,0 +1,41 @@
+/** @addtogroup uart_file
+
+@author @htmlonly &copy; @endhtmlonly 2018 
+Gerrit Maus <funk@maus.xyz>
+Luong Le <novercy@live.com>
+
+This library supports the UART in the NRF52 series.
+
+*/
+
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2018 Gerrit Maus <funk@maus.xyz>
+ * Copyright (C) 2018 Luong Le <novercy@live.com>
+ *
+ * This library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**@{*/
+
+#include <libopencm3/nrf5/uart.h>
+
+/*---------------------------------------------------------------------------*/
```

Comment:
please don't include empty files like this, there's no point, just leave themuntil you need them.